src/bin/sonar-scanner

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env node
-var scan = require('../index').cli
+var scan = require('../index').scan;
```

Comment:
```suggestion
const scan = require('../index').scan;
```

Reply:
ok, didn't change any var, not sure if we are supposed to support old js. I guess given that we require node >14 we can change them all